exercises/simple-cipher/canonical-data.json

```diff
@@ -16,13 +17,13 @@
           "input": {
             "plaintext": "aaaaaaaaaa"
           },
-          "expected": "cipher.key"
+          "expected": "cipher.key.substring(0, plaintext.length)"
         },
         {
           "description": "Can decode",
           "property": "decode",
           "input": {
-            "ciphertext": "cipher.key"
+            "ciphertext": "cipher.key.substring(0, expected.length)"
```

Comment:
I agree, but I think we all realize that we are really just trying to communicate that length of "another" string should be used here.  I saw this as a way to avoid the need to make two changes if someone later determined that fewer or more characters were needed.  To me, this is a kin to short hand notation and not necessarily code.